src/test/java/com/softserveinc/ita/ProtocolTest.java

```diff
@@ -47,4 +48,37 @@ public void verifyDatePickersOptionsChoosing() {
                 .as("When both date pickers are filled correctly search button should be enabled")
                 .isTrue();
     }
+
+    @Test
+    public void verifyDatePickersArrowsWork() {
+
+        LocalDate currentDate = LocalDate.now();
```

Comment:
create util class for dates DateTimeUtil

and create method there getCurrentDate()